bundles/org.openhab.core.io.net/src/main/java/org/openhab/core/io/net/http/PEMTrustManager.java

```diff
@@ -62,15 +62,17 @@ public final class PEMTrustManager extends X509ExtendedTrustManager {
      * <code>"-----END CERTIFICATE-----"</code>. The base 64 encoded certificate information are placed in between.
      *
      * @param pemCert the PEM certificate
-     * @throws CertificateException
+     * @throws CertificateInstantiationException
+     * @throws CertificateParsingException
      */
     public PEMTrustManager(String pemCert) throws CertificateException {
         if (!pemCert.isBlank() && pemCert.startsWith(BEGIN_CERT)) {
             try (InputStream certInputStream = new ByteArrayInputStream(pemCert.getBytes(StandardCharsets.UTF_8))) {
                 trustedCert = (X509Certificate) CertificateFactory.getInstance("X.509")
                         .generateCertificate(certInputStream);
             } catch (IOException e) {
-                throw new CertificateException(e);
+                LoggerFactory.getLogger(PEMTrustManager.class).debug("An IOException occurred: {}", e.getMessage());
```

Comment:
Why is the logger not an instance field? Otherwise LGTM

Reply:
Done.